packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -204,7 +205,7 @@ export class AccordionItem implements ConditionalSlotComponent {
             </div>
             {this.renderActionsEnd()}
           </div>
-          <div class={CSS.content}>
+          <div aria-live="polite" class={CSS.content} id={IDS.slotContainer} role="region">
```

Comment:
I thought we were still trying to use unique ids for each component instance? Even though its in shadowDOM.

Reply:
Yeah, I think we were following this pattern since the early days because we didn't have shadow DOM enabled consistently. If AT picks this up smoothly, we can keep things simple for the only-need-unique-ID-in-the-component's-shadow-DOM case.

Reply:
Hm, yeah, I'm not sure if this is an unique id issue and/or shadow DOM issue but JAWS isn't providing context with the current solution. Maybe we can try out an unique id?

Reply:
@geospatialem I made some tweaks. Could you test again?